src/disco/components/Addon.js

```diff
@@ -96,8 +110,9 @@ export default class Addon extends React.Component {
     return (
       <div className={addonClasses}>
         {this.getThemeImage()}
+        {this.getLogo()}
         <div className="content">
```

Comment:
Likewise, classes like `content` are probably going to get us into trouble later.